### PR TITLE
Don't remove hooks in tests

### PR DESCRIPTION
**Pull request by bserdar**

### CODE REVIEW

File: publish-hook/src/test/java/com/redhat/lightblue/hook/publish/BasePublishHookTest.java [hunk 1]
```diff
@@ -30,12 +33,20 @@ public BasePublishHookTest() throws Exception {
...
+    
     @Before
     public void before() throws UnknownHostException {
         cleanupMongoCollections("country", "esbEvents");
     }
 
     protected void insertCountry() throws Exception {
+        com.mongodb.DBCursor cursor=mongoServer.getConnection().getDB("testdb").getCollection("metadata").find();
+        while(cursor.hasNext())
+            System.out.println(cursor.next());
```
Comment (dcrissman) [inline publish-hook/src/test/java/com/redhat/lightblue/hook/publish/BasePublishHookTest.java]: I don't understand what this is doing?

Comment (bserdar) [inline publish-hook/src/test/java/com/redhat/lightblue/hook/publish/BasePublishHookTest.java]: Printing metadata in the DB. That's how I figured out hooks are being
removed: there are no hooks in the metadata in the db.

Comment (dcrissman) [inline publish-hook/src/test/java/com/redhat/lightblue/hook/publish/BasePublishHookTest.java]: I see. I would like for us to remove this block though. It assumes the use of testdb, which is the default but not a guarantee. Also, when debugging, you can place a break point in the asserts and connect to the in-memory mongo directly using `mongo localhost:27777`.

Comment (bserdar) [inline publish-hook/src/test/java/com/redhat/lightblue/hook/publish/BasePublishHookTest.java]: removed.

File: publish-hook/src/test/resources/simplelogger.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+org.slf4j.simpleLogger.log.com.redhat.lightblue=DEBUG
+org.slf4j.simpleLogger.log.com.redhat.lightblue.util.Error=ERROR
+org.slf4j.simpleLogger.defaultLogLevel=ERROR
```
Comment (dcrissman) [inline publish-hook/src/test/resources/simplelogger.properties:3]: Looks like this is producing too many lines for travis. Is it worth removing, or at least upping the error levels?

Comment (bserdar) [inline publish-hook/src/test/resources/simplelogger.properties:3]: Dont' remove, i'll change them to error.

Comment (bserdar) [inline publish-hook/src/test/resources/simplelogger.properties:3]: Done.

**Pull request merged**